src/app/visual/avatar/avatar-visual.component.html

```diff
@@ -9,3 +13,58 @@
 <div id="screenshot-avatar-initials" style="display: inline-block;">
   <sky-avatar [name]="name"></sky-avatar>
 </div>
+
+<div id="screenshot-avatar-sizes" style="display: inline-block;">
```

Comment:
There's a few attributes in this file that need to be on new lines.

Reply:
Fixed.